lua/sol.lua

```diff
@@ -34,7 +34,7 @@ return function (net_conf, sol_lcores)
 
 	sol_conf.log_ratelimit_interval_ms = log_ratelimit_interval_ms
 	sol_conf.log_ratelimit_burst = log_ratelimit_burst
-	sol_conf.pri_req_max_len = pri_req_max_len
+	sol_conf.pri_req_max_len__ = pri_req_max_len / #sol_lcores
```

Comment:
1. We should keep the meaning of `pri_req_max_len` as a value per SOL instance; otherwise, the value of `pri_req_max_len__` may silently become very small as the number of instances grows. We had a similar problem with GK instances while Cody was working on the scanning of the flow table. So just drop `/ #sol_lcores` and let's adjust the code accordingly.

2. Let's also change the current value of `pri_req_max_len` from 1024 to 512 since our deployment will be with two NUMA nodes.